FEM/fem_ele_vec.cpp

```diff
@@ -1325,6 +1298,9 @@ void CFiniteElementVec::GlobalAssembly_Stiffness()
 	if (act_nodes_h != nnodesHQ)
 		non_ghost_flag = -1;
 
+	petsc_group::PETScLinearSolver* eqs = pcs->eqs_new;
+
+	const PetscInt zero_id_alias = -eqs->Size();
```

Comment:
Because there is a node ID of 1, we cannot use -1 instead of 1. That means If the nodes with ID of 1 and the node with ID of 0 are ghost, there are two nodes have the same ID of -1.